packages/amplify-migration-tests/src/configure/index.ts

```diff
@@ -0,0 +1,51 @@
+import * as nexpect from 'nexpect';
+
+import { getCLIPath, isCI } from '../utils';
+type AmplifyConfiguration = {
+  accessKeyId: string;
+  secretAccessKey: string;
+  profileName?: string;
+};
+
+const defaultSettings = {
+  profileName: 'amplify-integ-test-user',
+  region: '\r',
+  userName: '\r',
+};
+
+const MANDATORY_PARAMS = ['accessKeyId', 'secretAccessKey'];
+export default function amplifyConfigure(settings: AmplifyConfiguration, verbose: Boolean = isCI() ? false : true) {
+  const s = { ...defaultSettings, ...settings };
+  const missingParam = MANDATORY_PARAMS.filter(p => !Object.keys(s).includes(p));
+  if (missingParam.length) {
+    throw new Error(`mandatory params ${missingParam.join(' ')} are missing`);
+  }
+
+  return new Promise((resolve, reject) => {
+    nexpect
+      .spawn(getCLIPath(), ['configure'], { stripColors: true, verbose })
+      .wait('Sign in to your AWS administrator account:')
+      .wait('Press Enter to continue')
+      .sendline('\r')
+      .wait('Specify the AWS Region')
+      .sendline('\r')
+      .wait('user name:')
+      .sendline('\r')
+      .wait('Press Enter to continue')
+      .sendline('\r')
+      .wait('accessKeyId')
```

Comment:
make sure to wrap the keys input inside a pauseRecording after you switch to amplify-e2d-core's `nspawn` like the following

      .pauseRecording()
      .sendLine(s.accessKeyId)
      .wait('secretAccessKey')
      .sendLine(s.secretAccessKey)
      .resumeRecording()